tools/defs.bzl

```diff
@@ -0,0 +1,61 @@
+# Copyright Google Inc. All Rights Reserved.
```

Comment:
name the file after the rule it exports (like you would in TS code)
`tools/ts_json_schema.bzl`